doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -607,344 +797,50 @@ Response code:
 
 + Response 204
 
+## Standard queryContext [/v2/queryContext]
 
-## Context Entity Subscriptions [/v2/entities/{entityId}/subscriptions]
-
-### List all subscriptions present in the system [GET /v2/subscriptions]
-
-+ Response 200
-
-        [
-            {
-                "id": "abcdefg",
-                "entity": {
-                    "id": "Bcn_Welt",
-                    "type": "Room"
-                },
-                "attributes": [
-                    "temperature"
-                ],
-                "callback": "http://localhost:1234",
-                "options": {
-                    "duration": "1M",
-                    "condition": "ONCHANGE",
-                    "throttling": "PT5S"
-                }
-            },
-             {
-                "id": "wyztrew",
-                "entity": {
-                    "pattern": "DC_*",
-                    "type": "Room"
-                },
-                "attributes": [
-                    "temperature"
-                ],
-                "callback": "http://localhost:1234"
-            }
-        ]
-
-### Get the details of a subscription [GET /v2/subscriptions/{subscriptionId}]
-
-+ Parameters
-    + subscriptionId: abcdefg (required, string) - ID of the suscription
-
-+ Response 200
-
-        {
-            "id": "abcdefg",
-            "entity": {
-                "id": "Bcn_Welt",
-                "type": "Room"
-            },
-            "attributes": [
-                "temperature"
-            ],
-            "callback": "http://localhost:1234",
-            "options": {
-                "duration": "1M",
-                "condition": "ONCHANGE",
-                "throttling": "PT5S"
-            }
-        }
-
-### Remove a suscription [DELETE /v2/subscriptions/{subscriptionId}]
-
-+ Parameters
-    + subscriptionId: abcdefg (required, string) - ID of the suscription
-
-+ Response 204
-
-
-### Create a new subscription [POST /v2/subscriptions]
+### queryContext [POST /v2/queryContest]
 
-+ Request (application/json)
+## Standard updateContext [/v2/updateContext]
```

Comment:
Three things to take into account:

* There isn't a "formal reference" for v1 operations in apiary (as fiware-ngsiv2-reference.apib is or will be). There is an apiary document for v1 operations, but is closer to a cookbook than to a formal reference (have a look to https://github.com/telefonicaid/fiware-orion/blob/develop/doc/apiary/orioncontextbroker.apib)

* Asumming there were such reference (we can create it), the operations could have slight diferences that make sense to have them also in this API v2 document. At least, the URL will change, e.g. from "/v1/queryContext" to "/v2/queryContext" given that each NGSI API version at URL level (i.e. "/v2") should be self-contained.

However, we don't need to close this in this PR. By the moment I would suggest to leave the placeholder (given that it is easier to remove things in the future than removing them now and adding them back if we change our mind :)